README.md

```diff
@@ -302,6 +302,11 @@ Each day I take one subject from the list below, read it cover to cover, take no
 - [DigitalMind's Artificial Intelligence resources](http://blog.digitalmind.io/post/artificial-intelligence-resources)
 - [Awesome Machine Learning](https://github.com/josephmisiti/awesome-machine-learning)
 - [CreativeAi's Machine Learning](http://www.creativeai.net/?cat%5B0%5D=machine-learning)
+- [ ] Machine Learning courses in Universities
+    - [Stanford](http://ai.stanford.edu/courses/)
+    - [Machine Learning Summer Schools](http://mlss.cc/)
+    - [Oxford](https://www.cs.ox.ac.uk/people/nando.defreitas/machinelearning/)
+    - [Cambridge](http://mlg.eng.cam.ac.uk/)
```

Comment:
We already had Machine Learning Summer Schools in the "Resource" section.
I think we should put [ ] before each university and move "Machine Learning courses in Universities" above "Flipboard Topics"